src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -930,6 +992,7 @@ bool clang_cpp_convertert::get_decl_ref(
   {
     msg.error(fmt::format(
       "Conversion of unsupported clang decl ref: {}", decl.getDeclKindName()));
+    decl.dumpColor();
```

Comment:
This will dump on stdout directly, could you use the same approach that is used for the c-frontend?

Reply:
ACK. 
will use `llvm::raw_os_ostream` for future clang AST node dumping.